### DISCUSSION TITLE
When do you plan to release 2.6.RC1

### DISCUSSION THREAD

**Discussion by yebolenko** [argoproj/argo-helm#1739] [Q&A] [answered]
Argo-cd project has released 2.6.RC1 more than week ago.
When do you plan to release 2.6.RC1 ?

**Comment by jmeridth** [comment, marked as answer]
We don't currently release helm charts for RC versions of app releases.

**Reply by yebolenko** [reply]
Jason, thank you for your response. Do you know when 2.6 release is planned?

**Reply by pdrastil** [reply]
It should be released in January if I remember correctly from upstream roadmap.
